worksheet_D/TicTacToe.py

```diff
@@ -0,0 +1,114 @@
+class OxoBoard:
+
+    def __init__(self,x,y):
+        self.x = x
+        self.y = y
+        self.board = {(0,0):0,(0,1):0,(0,2):0,(1,0):0,(2,0):0,(1,1):0,(2,2):0,(1,2):0,(2,1):0}
+
+    def get_square(self, x, y):
+        return self.board[x,y]
+
+    def set_square(self, x, y, full):
+        if self.board[x,y] == 0:
+            self.board[x,y] = full
+            return True
+        else:
+            return False
+
+    def is_board_full(self):
+        spaces = 9
+        for i in xrange(0,3):
+            for j in xrange(0,3):
+                if self.board[i, j] > 0:
+                    spaces -= 1
+                if spaces == 0:
+                    return True
+        return False
+
+    def get_winner(self, player_number):

```

Comment:
I could not test your code as the additional parameters here and in `__init__` cause my test script to fail. Do not deviate from the specification!